src/Ledger/Tally.lagda

```diff
@@ -24,8 +27,16 @@ open import Ledger.Address Network KeyHash ScriptHash
 \end{code}
 \begin{figure*}[h]
 \begin{code}
+data Voter : Set where
```

Comment:
This is a strange type. I think if we have a type like this at all, it should be isomorphic to `VDeleg × GovRole`.

Reply:
I was considering that, but `noConfidenceRep , CC` sounds a bit odd.